packages/cli/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Maybe we've to ignore this file? Is it related somehow with the build process?

Reply:
I'm not sure where it comes from. Do you know?

Reply:
Looking what it is: https://editorconfig.org it seems valuable and should reside into the repo, since it aims to make development consistent.